index.html

```diff
@@ -3,15 +3,186 @@
+                 <p class="para">Benkler studies commons-based peer production, and published his seminal book, The Wealth of Networks in 2006</p>
+                </div>
+               </div> 
+               <div class="speakers">
+                <img src="images/speak.png" alt="speaker">  
+                <div class="text-align">
+                 <h4>Kilnam Chon</h4>
+                 <h6>Berkman Professor of Entrepreneurial Legal Studies at Harvard Law School</h6>
+                 <p class="para">Benkler studies commons-based peer production, and published his seminal book, The Wealth of Networks in 2006</p>
+                </div>
+               </div>
+            </article> 
+            <article>
+               <div class="speakers">
+                <img src="images/speaker_01 1.png" alt="speaker">  
+                <div class="text-align">
+                 <h4>Yochai Benkler</h4>
+                 <h6>Berkman Professor of Entrepreneurial Legal Studies at Harvard Law School</h6>
+                 <p class="para">Benkler studies commons-based peer production, and published his seminal book, The Wealth of Networks in 2006</p>
+                </div>
+               </div> 
+            
+               <div class="speakers">
+                <img src="images/spea.png" alt="speaker">  
+                <div class="text-align">
+                 <h4>SohYeong Noh</h4>
+                 <h6>Berkman Professor of Entrepreneurial Legal Studies at Harvard Law School</h6>
+                 <p class="para">Benkler studies commons-based peer production, and published his seminal book, The Wealth of Networks in 2006</p>
+                </div>
+               </div> 
+        </article> -->
+        <button id="btn-more" class="btn-expand" type="button">
+            MORE <img src="images/arrow_down 1.png" alt="" id="down">
+          </button>
+          <button id="btn-hide"  class="btn-expand" type="button">
+            HIDE <i class="fa-solid fa-chevron-up" id="up"></i>
+          </button>
+      </section>
 
+      <footer>
+        <h1>Partner</h1>
+        <ul class="line-1">
+            <li><img src="images/partner/mozilla 1.png" alt=""></li>
+            <li><img src="images/partner/googlekorea 1.png" alt=""></li>
+            <li><img src="images/partner/naver 1.png" alt=""></li>
+            <li><img src="images/partner/daumkakaologo 1.png" alt=""></li>
+            <li><img src="images/partner/airbnb 1.png" alt=""></li>
+        </ul>
 
-   <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/js/bootstrap.bundle.min.js" integrity="sha384-MrcW6ZMFYlzcLA8Nl+NtUVF0sA7MsXsP1UyJoMp4YLEuNSfAP+JcXn/tWtIaxVXM" crossorigin="anonymous"></script>
+        <div class="copyrigth">
+            <img src="images/Market (1).png" alt="">
+            <p>2015 Creative Commons Korea. Some Rights Reserved.</p>
+        </div>
+    </footer>
```

Comment:
- [ ] Kindly make sure that you use There's at least 1, but no more than 1, `h1` tag in each page, So Kindly **I recommend you to use `h3` tag for the Partner heading and make the partner as` section` tag instead of the footer page** , The Footer page should only contains **a logo aligned to the left and text aligned to the right.** Kindly Fix the issues in order to keep up the HTML hierarchy  and best practices Just like the below code snippets. 
```
<section  id="Partners">
        <div >
          <h3 >Partners</h3>
        </div>
        <ul >
          <li>
            <img >
          </li>
          <li>
            <img >
          </li>
          <li>
            <img >
          </li>
          <li>
            <img >
          </li>
          <li>
            <img >
          </li>
        </ul>
    </section>
 

   <footer >
        <div >
            <img >
        </div>
        <div  >
          <p></p>
        </div>
      </div>
   </footer>
```